corehq/messaging/smsbackends/airtel_tcl/tests/test_airtel_tcl.py

```diff
@@ -110,14 +110,17 @@ def test_get_url(self):
         )
 
     def test_get_phone_number(self):
-        self.assertEqual(AirtelTCLBackend.get_phone_number('+910123456789'), '0123456789')
-        self.assertEqual(AirtelTCLBackend.get_phone_number('910123456789'), '0123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('+918123456789'), '8123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('916123456789'), '6123456789')
 
         with self.assertRaises(InvalidDestinationNumber):
-            AirtelTCLBackend.get_phone_number('+999123456')
+            AirtelTCLBackend.get_phone_number('+918123455')
```

Comment:
why was this test updated?

Reply:
I ask this because the previous checks should still be valid. The new ones we added
1. length should be 10
2. should start with 6, 7, 8, 9
should not need to change any earlier tests. Whatever was invalid earlier, should still be invalid.

Reply:
I added one test where length is less than 10, one where it does not start with 91 and one where the first digit is less than 6. Previous checks are still invalid

Reply:
Detailed explanation Offline,

Mainly, overall, since we added "more" validation on the one that already existed, the old tests should still be "valid", i mean that they should still make sense and should not need to updated.
for places where you changed the numbers to start with 6-9, make sense because we don't consider the numbers set earlier, valid anymore